libshaderc/include/shaderc.hpp

```diff
@@ -104,6 +104,23 @@ class CompileOptions {
     AddMacroDefinition(name.c_str(), value.c_str());
   }
 
+  // Set the compiler to emit a disassembly text instead of a binary. In
```

Comment:
Today I feel like asking for the blurbs to use standard style. So:
"Sets the compiler mode to emit disassembly text instead of a binary."   Ok, to just do for the C++ API.  We can do the C API another time.